src/ReactLeafletMap/index.js

```diff
@@ -29,6 +29,7 @@ class ReactLeafletMap extends React.PureComponent {
         onClick: PropTypes.func,
         onLayerClick: PropTypes.func,
         onViewChange: PropTypes.func,
+		resources: PropTypes.object,
```

Comment:
indetation

Reply:
It'll be fixed in the next PR when our code styling rules are applied